src/lightning/data/streaming/cache.py

```diff
@@ -30,7 +30,7 @@
 
 logger = logging.Logger(__name__)
 
-if _LIGHTNING_CLOUD_GREATER_EQUAL_0_5_50:
+if _LIGHTNING_CLOUD_GREATER_EQUAL_0_5_51:
```

Comment:
Since the minimum version is declared in the requirements, we should remove these conditional imports. It doesn't make sense to change the code in several places every time we bump the version.

Reply:
done !